src/app/components/bibliography/bibliography.component.ts

```diff
@@ -0,0 +1,163 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { BibliographicCitation, BibliographyParserService } from '../../services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography',
+  templateUrl: './bibliography.component.html',
+  styleUrls: ['./bibliography.component.scss'],
+})
+export class BibliographyComponent implements OnDestroy {
+
+  public biblCits: BibliographicCitation[] = [];
+  private subscription: Subscription;
+  public isComplete: boolean;
+  selectedAlphOrder: 'Ascendent';
+  selectedField: 'Author';
+  selectedStyle: 'Chicago';
+  features: string[] = ['Author', 'Title', 'Date'];
+  styles: string[] = ['Chicago', 'APA'];
```

Comment:
These should probably go inside conf? @ChiaraDipi

Reply:
Yes, the user-editor should be able to decide which style(s) make available in his edition.

Reply:
Should I also put `Chicago` and `APA` in the configuration file?